vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -845,6 +864,11 @@ We will try to install .NET, but are unlikely to be able to connect to the serve
         ensureDependenciesRegistration,
         reportIssueRegistration,
         ...eventStreamObservers);
+
+    if (showResetDataCommand)
+    {
+        vsCodeContext.subscriptions.push(resetDataPublicRegistration);

```

Comment:
Is there an intended order for these actions?

Reply:
Good question -- no, the order shouldn't matter for the behavior of the application